addons/website_livechat/static/src/web/livechat_channel_info_list_patch.xml

```diff
@@ -3,6 +3,15 @@
     <t t-name="website_livechat.LivechatChannelInfoList" t-inherit="im_livechat.LivechatChannelInfoList" t-inherit-mode="extension">
         <xpath expr="//t[@t-name='extra_infos']" position="inside">
             <t t-set="visitor" t-value="props.thread.livechat_visitor_id"/>
+            <div class="d-flex gap-3 pt-3">
+                <span title="Language" t-if="visitor?.lang_id">
```

Comment:
May need to make this conditional as it could be empty with padding. Adding unnecessary spacing.